plugins/node/opentelemetry-instrumentation-graphql/src/enums/AttributeNames.ts

```diff
@@ -19,7 +19,8 @@ export enum AttributeNames {
   FIELD_NAME = 'graphql.field.name',
   FIELD_PATH = 'graphql.field.path',
   FIELD_TYPE = 'graphql.field.type',
-  OPERATION = 'graphql.operation.name',
+  OPERATION_TYPE = 'graphql.operation.type',
```

Comment:
Is this specified?

Reply:
I don't understand what you mean @dyladan.

Reply:
@mentos1386 I suspect @dyladan is asking whether there are OpenTelemetry semantic conventions for GraphQL trace data. (see here for what that means if you're unsure: https://github.com/open-telemetry/opentelemetry-js-contrib/issues/705#issuecomment-952998047)

If so, the answer is "no", but progress toward having some is occurring here: https://github.com/open-telemetry/opentelemetry-specification/issues/1670

Reply:
Yes @spencerwilson is correct that's exactly what I meant. In general we hesitate to add/change attributes without semantic conventions, but in this case it seems like the spec is moving quite slowly.